whois-api/src/test/java/net/ripe/db/whois/api/log/JettyRequestLogTestIntegration.java

```diff
@@ -43,7 +49,7 @@
             "mnt-by:        OWNER-MNT\n" +
             "source:        TEST");
 
-    @Value("var${jvmId:}/log/jetty")
+    @Value("target/log/jetty")
```

Comment:
The jvmId is important for separating logging for tests running in parallel, can you re-add it?